tests/baseline/cases/staged-simple-cloud.yml

```diff
@@ -0,0 +1,193 @@
+# A pushsource library testcase.
+#
+# This file was generated from a template.
+# To regenerate, run test_baseline.py with PUSHSOURCE_UPDATE_BASELINE=1.
+
+# URL of Source to test.
+url: "staged:{{ src_dir }}/tests/staged/data/simple_cloud"
+
+# Push items generated from above.
+items:
+- AmiPushItem:
+    access_endpoint_url: null
+    billing_codes: null
+    boot_mode: hybrid
+    build: null
+    build_info:
+      id: null
+      name: rhel-ec2
+      release: '1'
+      version: '9.4'
+    description: build2 sample
+    dest: []
+    ena_support: null
+    image_id: null
+    marketplace_entity_type: null
+    marketplace_name: null
+    marketplace_title_template: null
+    md5sum: null
+    name: something1.raw.xz
+    origin: {{ src_dir }}/tests/staged/data/simple_cloud/starmap/CLOUD_IMAGES/build2
+    public_image: null
+    recommended_instance_type: null
+    region: null
+    release: null
+    release_notes: null
+    root_device: null
+    scanning_port: null
+    security_groups: []
+    sha256sum: null
```

Comment:
Something I've just noticed: in the resources.yaml there are no checksums.  That's pretty unusual.  For most kinds of push items supported today, there is some form of support for the expected checksums to be passed into pushsource library, allowing the content to be verified while it's loaded.

Maybe that should be the case here as well, with resources.yaml having a sha256sum which is then propagated into the push item.

Can you discuss in your team whether this should be added? I can't say it's strictly required, but if omitted some of the consequences might be:

- in case something unexpected/corrupted gets pushed, it will be ambiguous which component was at fault
- in case you want to reliably test whether something has already been uploaded to a remote service, you will have to read the entire image file to calculate its checksum in order to do that check.  You can't cheaply compare local and remote checksums, as you have no local checksums.

Reply:
Discussed it with the team and it makes sense. So I added it.